modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2468,4 +2468,23 @@ private <T> CompletableFuture<T> orStopManagerFuture(CompletableFuture<T> future
 
         return anyOf(future, stopManagerFuture).thenApply(o -> (T) o);
     }
+
+    /** Internal event. */
+    private static class DestroyTableEvent {
```

Comment:
Please use package private class.

Reply:
Why? 
This class can't be used outside the parent.